src/lib/blockchain_snark/tests/test_blockchain_snark.ml

```diff
@@ -0,0 +1,27 @@
+open Core_kernel
+
+let regtest_digest_blockchain_snark () =
+  let expected_digest = "36786c300e37c2a2f1341ad6374aa113" in
```

Comment:
What happen if we hardfork, or is there any operation that will affect this digest? 

If so, could we track this in a constants module or something?

Reply:
It's a regression test, therefore it's expected to fail if it changes.